weblate/api/serializers.py

```diff
@@ -1092,6 +1092,9 @@ class UnitSerializer(serializers.ModelSerializer[Unit]):
         ),
         strip_parts=1,
     )
+    language_code = serializers.CharField(
+        source="translation.language_code", read_only=True
```

Comment:
Using a standardized language code instead of the one used in the filename is probably more useful:

```suggestion
        source="translation.language.code", read_only=True
```

Reply:
done.